### PR TITLE
First draft of 1.34.1 announcement

### PR DESCRIPTION
**Pull request by Centril**
r? @rust-lang/release @rust-lang/core

Written in collaboration with @pietroalbini.

### CODE REVIEW

File: posts/2019-04-25-Rust-1.34.1.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+```
+
+## What's new in rustup 1.18.1
+
+[a regression]: https://github.com/rust-lang/rustup.rs/issues/1794
+[full release notes]: https://github.com/rust-lang/rustup.rs/blob/stable/CHANGELOG.md#1181---2019-04-25
+
+A recent rustup release, 1.18.0, introduced [a regression] that prevented installing Rust through the shell script on older platforms.
+A patch was released that fixes the issue, avoiding to force TLSv2 on the platforms that don't support it.
```
Comment (jonas-schievink) [inline posts/2019-04-25-Rust-1.34.1.md]:
```suggestion
A patch was released that fixes the issue, avoiding to force TLS v1.2 on the platforms that don't support it.
```

File: posts/2019-04-25-Rust-1.34.1.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+The Rust team is happy to announce a new version of Rust, 1.34.1, and a new version of rustup, 1.18.1.
+Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup,
+getting Rust 1.34.1 and rustup 1.18.1 is as easy as:
+
+```console
+$ rustup update stable
```
Comment (jonas-schievink) [inline posts/2019-04-25-Rust-1.34.1.md]: I don't think this updates rustup, for that you need to run without arguments.

Maybe add another line with `rustup self update`?

Comment (Centril) [reply posts/2019-04-25-Rust-1.34.1.md]: I thought the same but @pietroalbini checked this.

Comment (kinnison) [reply posts/2019-04-25-Rust-1.34.1.md]: Rustup automatically self-updates on any update unless `--no-self-update` is passed :D  (Feature as of 1.17.0)

Comment (jonas-schievink) [reply posts/2019-04-25-Rust-1.34.1.md]: Ah I see, the rustup 1.17.0 CLI help doesn't make this very clear

**Pull request merged**